librz/cons/dietline.c

```diff
@@ -1209,6 +1209,7 @@ static void __vi_mode(void) {
 			break;
 		case 'c':
 			I.vi_mode = INSERT_MODE; // goto insert mode
+			break;
```

Comment:
did not check this... is this really right as well?

Reply:
Yes, was wrong as well. Added a fall-through comment to indicated that.